src/Smalot/PdfParser/PDFObject.php

```diff
@@ -556,9 +556,13 @@ public function parseDictionary(string $dictionary): array
     }
 
     /**
-     * getText() leverages getTextArray() to get the content of the
-     * document, setting the addPositionWhitespace flag to true so
-     * whitespace is inserted in a logical way for reading by humans.
+     * Returns the text content of a PDF as a string. Attempts to add
+     * whitespace for spacing and line-breaks where appropriate.
+     *
+     * @internal getText() leverages getTextArray() to get the content
+     * of the document, setting the addPositionWhitespace flag to true
+     * so whitespace is inserted in a logical way for reading by
+     * humans
```

Comment:
```suggestion
     * getText() leverages getTextArray() to get the content
     * of the document, setting the addPositionWhitespace flag to true
     * so whitespace is inserted in a logical way for reading by
     * humans
     *
     * @internal
```

Reply:
This function should not be @internal. I was just using it to mark text as only appearing in the internal documentation, at least that's the impression I got from the phpdoc site I was looking at. I'll switch to the tagging method instead.